monitoring/messagesigning/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flask==1.1.2
```

Comment:
This file should not be required anymore since dependencies are provided by the [monitorlib requirements](https://github.com/interuss/dss/blob/master/monitoring/monitorlib/requirements.txt).